scene/gui/button.h

```diff
@@ -69,6 +68,9 @@ class Button : public BaseButton {
 	void set_icon(const Ref<Texture> &p_icon);
 	Ref<Texture> get_icon() const;
 
+	void set_expand_icon(bool p_expand_icon);
+	bool is_expand_icon() const;
```

Comment:
Should it be `is_expanding_icon`? Bool getters are not fully consistent so far in the API, we'll have to pick a style and maybe review the API for 4.0.

Reply:
I could change all instances to "expandable_icon", so it works for both `set` and `is`.
We definitely need a single style for names, as there are `bool`s that even use `get` instead of `is`.